source/content/object-cache.md

```diff
@@ -537,7 +543,7 @@ activerehashing yes
 
 Note that the `maxmemory` value will vary based on plan level.
 
-### If Redis hits the upper limit of memory usage, is this logged on Pantheon?
+### If Object Cache hits the upper limit of memory usage, is this logged on Pantheon?
 
 Yes. There is a `redis.log` file that is available on the Redis container for each environment.
```

Comment:
flagging, but assuming there's no change planned to the name of the log

Reply:
Correct, no change planned of which I'm aware, thanks.